test/TritonGPU/tf32x3-matmul.mlir

```diff
@@ -0,0 +1,54 @@
+// RUN: triton-opt %s -split-input-file -tritongpu-F32DotTC -canonicalize  | FileCheck %s --check-prefixes=CHECK 
+
+// CHECK:     %[[DOT1:.*]] = tt.dot %[[LHS_LOW:.*]], %[[RHS_HIGH:.*]], %cst_1, inputPrecision = tf32 : tensor<32x32xf32> * tensor<32x32xf32> -> tensor<32x32xf32>
+// CHECK:     %[[DOT2:.*]] = tt.dot %[[LHS_HIGH:.*]], %[[RHS_LOW:.*]], %[[DOT1]], inputPrecision = tf32 : tensor<32x32xf32> * tensor<32x32xf32> -> tensor<32x32xf32>
+// CHECK:     %[[CMP:.*]] = arith.cmpf uno, %[[DOT2]], %[[DOT2]] : tensor<32x32xf32>
+// CHECK:     %[[MASKED:.*]] = arith.select %[[CMP]], %cst_1, %[[DOT2]] : tensor<32x32xi1>, tensor<32x32xf32>
+// CHECK:     %[[RESULT:.*]] = tt.dot %[[LHS_HIGH]], %[[RHS_HIGH]], %[[MASKED]], inputPrecision = tf32 : tensor<32x32xf32> * tensor<32x32xf32> -> tensor<32x32xf32>
+
+module {
```

Comment:
please minimize the test

Reply:
you should be able to reduce it further. Yhe pattern only consider a single dot op so you should have a function with just one operation. 
It should be easy to find examples of lit tests written that way

Reply:
done